[2023-07-20T15:00:13.412+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ETL_stock_data_google_DAG.transform_load_data manual__2023-07-20T15:00:08.735364+00:00 [queued]>
[2023-07-20T15:00:13.528+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ETL_stock_data_google_DAG.transform_load_data manual__2023-07-20T15:00:08.735364+00:00 [queued]>
[2023-07-20T15:00:13.529+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-07-20T15:00:13.547+0000] {taskinstance.py:1327} INFO - Executing <Task(BashOperator): transform_load_data> on 2023-07-20 15:00:08.735364+00:00
[2023-07-20T15:00:13.554+0000] {standard_task_runner.py:57} INFO - Started process 251 to run task
[2023-07-20T15:00:13.559+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'ETL_stock_data_google_DAG', 'transform_load_data', 'manual__2023-07-20T15:00:08.735364+00:00', '--job-id', '102', '--raw', '--subdir', 'DAGS_FOLDER/ETL_stock_data_google_DAG.py', '--cfg-path', '/tmp/tmpr5b6r92y']
[2023-07-20T15:00:13.561+0000] {standard_task_runner.py:85} INFO - Job 102: Subtask transform_load_data
[2023-07-20T15:00:13.617+0000] {task_command.py:410} INFO - Running <TaskInstance: ETL_stock_data_google_DAG.transform_load_data manual__2023-07-20T15:00:08.735364+00:00 [running]> on host 90317fb99301
[2023-07-20T15:00:13.711+0000] {taskinstance.py:1545} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='fvinciguerra' AIRFLOW_CTX_DAG_ID='ETL_stock_data_google_DAG' AIRFLOW_CTX_TASK_ID='transform_load_data' AIRFLOW_CTX_EXECUTION_DATE='2023-07-20T15:00:08.735364+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-07-20T15:00:08.735364+00:00'
[2023-07-20T15:00:13.713+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-07-20T15:00:13.714+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'python /opt/***/dags/pyscripts/TransformationLoad_stock_data_google.py']
[2023-07-20T15:00:13.726+0000] {subprocess.py:86} INFO - Output:
[2023-07-20T15:00:15.062+0000] {subprocess.py:93} INFO - Traceback (most recent call last):
[2023-07-20T15:00:15.063+0000] {subprocess.py:93} INFO -   File "/opt/***/dags/pyscripts/TransformationLoad_stock_data_google.py", line 5, in <module>
[2023-07-20T15:00:15.063+0000] {subprocess.py:93} INFO -     from Parametros_conexiónnnnnn import host, database, port, user, password
[2023-07-20T15:00:15.064+0000] {subprocess.py:93} INFO - ModuleNotFoundError: No module named 'Parametros_conexiónnnnnn'
[2023-07-20T15:00:15.134+0000] {subprocess.py:97} INFO - Command exited with return code 1
[2023-07-20T15:00:15.145+0000] {taskinstance.py:1824} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/bash.py", line 210, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2023-07-20T15:00:15.148+0000] {taskinstance.py:1345} INFO - Marking task as FAILED. dag_id=ETL_stock_data_google_DAG, task_id=transform_load_data, execution_date=20230720T150008, start_date=20230720T150013, end_date=20230720T150015
[2023-07-20T15:00:15.176+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 102 for task transform_load_data (Bash command failed. The command returned a non-zero exit code 1.; 251)
[2023-07-20T15:00:15.221+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 1
[2023-07-20T15:00:15.248+0000] {taskinstance.py:2653} INFO - 1 downstream tasks scheduled from follow-on schedule check
